mlir/lib/Dialect/Rock/Transforms/BlockwiseGemmToThreadwise.cpp

```diff
@@ -412,163 +402,183 @@ struct BlockwiseGemmV2RewritePattern
 
     int64_t KPerThread = IsKReduction ? K / inputSpansPerMfmaIn : K;
 
+    auto outerLoopM = b.create<AffineForOp>(loc, 0, mRepeats);
+    auto olmb = ConversionPatternRewriter::atBlockBegin(outerLoopM.getBody(),
+                                                        b.getListener());
+    auto olmiv = outerLoopM.getInductionVar();
+    auto mOffset = olmb.create<AddIOp>(
+        loc, aBase, olmb.create<MulIOp>(loc, mPerMfmaGroupConstantOp, olmiv));
+
+    auto outerLoopN = olmb.create<AffineForOp>(loc, 0, nRepeats);
```

Comment:
Also, is it possible to make these `AffineForOp` to be `transforming_for`? I know the code was already in that way, but I am wondering if there is any technicality I am missing on why we can't use `transforming_for`. It would be beneficial also because it does have unrolls,etc..

Reply:
This all needs to be converted to transforming for and coordinate transforms. Right now it stays at a polished prototype level for performance evaluation.